### ISSUE TITLE
Develop Estimation/Sizing Best Practices

### ISSUE THREAD

**Issue by bensternthal** [mozmeao/agile-task-force#22]
### Description

Integrate learnings from AgileAgency around estimation into our documentation.

**Comment by MaciekPluta** [comment]
As discussed on 7/12, I wanted to share notes on how my teams estimate. 

TL;DR
We estimate/size **user stories** using t-shirt size method with days/points assigned to sizes.
 S (1 day), M (3 day) and Large (5 days). We use "ideal time" and take into account all of the team members time needed to complete user story. We use the data to be able to tell if we (as a team) has under or overcommitted. 

Details:
We use this system to size user stories:
3 size t-shirts: S, M and L. Small is 1 day, Medium 3 days and Large 5 days, of uninterrupted work by all team members. For example, if I need a day and Jenny needs 2 days to complete a user story, this would be a Medium (3). If an agency will spend another 5 days (after Jenny and I are done), this effort is disregarded, as it's an outside of the team. 

When do we estimate?
During 2nd day of the sprint planning, when the work has been broken down and tasks have been defined. 

Who estimates? 
All of the team members

What then? 
It's summed up to see the number of points the team committed to in a sprint to be able to tell if we have under or over committed. This is then used to prioritize the stories by the TL and the PM. 

Velocity tracking
I track velocity in a separate tab the WiP file

**Comment by bensternthal** [comment]
Moved to trello!
